### PR TITLE
chore: Update Sentry perf tracing to ignore ping

### PR DESCRIPTION
**Pull request by Nimphal**
Make sure sentry doesn't pick up 'ping' endpoint for performance sampling

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -82,7 +82,21 @@ if (config.SENTRY.DSN) {
           ]
         : [],
     // Half of all requests will be used for performance sampling
-    tracesSampleRate: config.SENTRY.ENVIRONMENT === 'production' ? 0.5 : 0,
+    tracesSampler: samplingContext => {
+      const transactionName =
```
Comment (Thource) [inline server.js:86]: I think we can do `samplingContext?.transactionContext?.name`

File: server.js [hunk 2]
```diff
@@ -82,7 +82,21 @@ if (config.SENTRY.DSN) {
...
+        samplingContext &&
+        samplingContext.transactionContext &&
+        samplingContext.transactionContext.name
+
+      if (config.SENTRY.ENVIRONMENT !== 'production') {
+        return 0
+      } else if (transactionName && transactionName.includes('ping')) {
+        return 0
+      } else {
```
Comment (Thource) [inline server.js:95]: We can get rid of the `else {}` :)

File: server.js (hunk 1 above)
Comment (Thource) [reply server.js:86]: Boo! ☹️

**Review by Thource — APPROVED** [review]

File: server.js (hunk 2 above)
Comment (Nimphal) [reply server.js:95]: It can, I'm sure this is not my last foray in this piece of code, so I'll clean it up next time :)

**Pull request merged**